about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/06d805300f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="about.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Globle summit</title>
+</head>
+<body>
+    <!-- Header section starts-->
+    <header>
+        <img id="menu-icon" class="menu-icon" src="./assets/mobile-menu.png" alt="">
+        <ul class="header-head">
+            <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+
+        <nav id="navbar" class="header-body">
+            <i id="mobile-menu-close-icon" class="mobile-menu-close-icon fa-solid fa-x"></i>
+            <a class="logo" href="index.html"><img src="./assets/Logopit_1684831391238.png" alt="logo"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Programe</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href=""><button class="header-btn">CC Campaign</button></a></li>
+            </ul>
+        </nav>
+    </header>
+    <!--Header section ends-->
+
+    <main>
+
+     <!-- Headline Section Starst Here -->
+     <section class="headline">
+        <h6 class="headline__greetings-heading">"Hello! Sharing world"</h6>
+        <h1  class="headline__main-heading">Global Tech Summit 2023</h1>
+        <p class="headline-description">The Global Summit brings together the community of  
+            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
```

Comment:
- [ ] As suggested by the reviewer, I would kindly advice too that you should reduce the width of your paragraph design and be sure it looks like what we have in the original design. 

>N.B: Kindly refer back to the previous comment made by the reviewer for more insight.